components/GeneralHeader.tsx

```diff
@@ -55,6 +58,9 @@ export default function GeneralHeader({
           )}
         </div>
       </div>
+      <div className="mx-auto w-[27.75rem] text-center text-xs text-gray-400">
```

Comment:
If subtitle can be nullish, please wrap in `{ subtitle && ... }`

Reply:
https://github.com/forbole/kastle/pull/24/commits/eeb9db586ca893bf012c17d5439c5f033ffc64b2
Fixed